main/main.c

```diff
@@ -49,6 +49,8 @@ uint64_t cycles_per_sec;
 uint64_t cycles_per_ms;
 uint64_t picosec_per_cycle;
 
+struct log_ratelimit_state log_ratelimit_states[RTE_MAX_LCORE];
+
```

Comment:
Move `log_ratelimit_states[]` to just before `rte_log_ratelimit()`, and make it `satic`.